core/src/main/java/com/novoda/noplayer/Listeners.java

```diff
@@ -30,9 +30,9 @@
 
     void removeBitrateChangedListener(Player.BitrateChangedListener bitrateChangedListener);
 
-    void addHeartbeatCallback(Heart.Heartbeat.Callback<Player> callback);
+    void addHeartbeatCallback(Player.HeartbeatCallback heartbeatCallback);
```

Comment:
Heart.Heartbeat.Callback has been moved to the Player to match the other listeners that the Player supports.